queue_job/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,43 @@
+* Using environment variables and command line:
+
+  * Adjust environment variables (optional):
+
+    - ``ODOO_QUEUE_JOB_CHANNELS=root:4`` or any other channels configuration. 
+      The default is ``root:1``
+
+    - if ``xmlrpc_port`` is not set: ``ODOO_QUEUE_JOB_PORT=8069``
+
+  * Start Odoo with ``--load=web,queue_job``
+    and ``--workers`` greater than 1. [1]_
+
+
+* Using the Odoo configuration file:
+
+.. code-block:: ini
+
+  [options]
+  (...)
+  workers = 4
+  server_wide_modules = web,queue_job
+
+  (...)
+  [queue_job]
+  channels = root:4
```

Comment:
That's out of scope and not new, but the basic configuration shown here can block all the HTTP workers. I think we should put `channels = root:2` or put more HTTP workers.

Reply:
Then it's good to change this at the same time I think

Reply:
Yes the queue_job documentation needs some love. I did not touch the content here.

Reply:
BTW with workers = 4 it would even be root:1, because one wkhtmlpdf report needs 2 workers so we need at least 3 free to generate a report and keep the UI working while the report runs.

Reply:
Right. I propose to merge this PR without any changes to the content.

Reply:
OK, merging now and waiting for later PRs for these considerations.